index.js

```diff
@@ -232,11 +225,42 @@ function linux (args, sudo, binary) {
         sink(args, proc.pid, folder, clock)
       )
     })
+  })
 
+  proc = spawn('sudo', [
```

Comment:
@davidmarkclements please note, that the diff looks larger then it is, because I basically just reordered code blocks. For git that's a deletion essentially it's just moving around things though.